### PR TITLE
react-tags: fill icon on hover for outlined InteractionTag

### PR DESCRIPTION
**Pull request by YuanboXue-Amber**
1. fill icon on hover for InteractionTag outlined variant:
image

2. update react-tag stories to use the same icon

### CODE REVIEW

**Comment by fabricteam** [comment]
## 📊 Bundle size report

| Package & Exports                                                                                                |   Baseline (minified/GZIP) |                          PR |                                                                                                                                                                                                   Change |
| :--------------------------------------------------------------------------------------------------------------- | -------------------------: | --------------------------: | -------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------: |
| <samp>react-tags-preview</samp> <br /> <abbr title='bundle-size/InteractionTag.fixture.js'>InteractionTag</abbr> | `33.855 kB`<br />`9.08 kB` | `34.145 kB`<br />`9.188 kB` | `290 B` <br />`108 B`  |

<details>
<summary>Unchanged fixtures</summary>

| Package & Exports                                                                                    |         Size (minified/GZIP) |
| ---------------------------------------------------------------------------------------------------- | ---------------------------: |
| <samp>react-tags-preview</samp> <br /> <abbr title='bundle-size/tag.fixture.js'>Tag</abbr>           |  `25.936 kB`<br />`8.483 kB` |
| <samp>react-tags-preview</samp> <br /> <abbr title='bundle-size/TagGroup.fixture.js'>TagGroup</abbr> | `69.142 kB`<br />`20.294 kB` |

</details>
<sub>🤖 This report was generated against <a href='https://github.com/microsoft/fluentui/commit/beef7217f6b332ce3339e81d5448135d9142c495'>beef7217f6b332ce3339e81d5448135d9142c495</a></sub>
<div id="bundleSizeComment9423"/>

File: packages/react-components/react-tags-preview/stories/InteractionTag/InteractionTagAppearance.stories.tsx [hunk 1]
```diff
@@ -15,13 +15,13 @@ export const Appearance = () => {
   const styles = useContainerStyles();
   return (
     <div className={styles.container}>
-      <InteractionTag icon={<CalendarMonth />} dismissible>
+      <InteractionTag icon={<CalendarMonthRegular />} dismissible>
```
Comment (ling1726) [inline packages/react-components/react-tags-preview/stories/InteractionTag/InteractionTagAppearance.stories.tsx:18]: why revert back to unbundled icons here?

File: packages/react-components/react-tags-preview/stories/InteractionTag/InteractionTagDisabled.stories.tsx [hunk 2]
```diff
@@ -15,19 +13,25 @@ export const Disabled = () => {
   const styles = useContainerStyles();
   return (
     <div className={styles.container}>
-      <InteractionTag disabled secondaryText="appearance=filled" icon={<CalendarMonth />} dismissible>
+      <InteractionTag disabled secondaryText="appearance=filled" icon={<CalendarMonthRegular />} dismissible>
```
Comment (ling1726) [inline packages/react-components/react-tags-preview/stories/InteractionTag/InteractionTagDisabled.stories.tsx:16]: shouldn't these all be bundled icons?

File: packages/react-components/react-tags-preview/stories/InteractionTag/InteractionTagAppearance.stories.tsx (hunk 1 above)
Comment (ling1726) [reply packages/react-components/react-tags-preview/stories/InteractionTag/InteractionTagAppearance.stories.tsx:18]: oh I see it's only outline appearance tags affected

**Review by ling1726 — APPROVED** [review]

File: packages/react-components/react-tags-preview/stories/InteractionTag/InteractionTagAppearance.stories.tsx (hunk 1 above)
Comment (YuanboXue-Amber) [reply packages/react-components/react-tags-preview/stories/InteractionTag/InteractionTagAppearance.stories.tsx:18]: > I'd still keep using bundled icons IMO, there's not much consequence to and it's what we encourage partners to do

Oh I see, I'll use bundled icon for the stories that I'm already bundle them :)

**Pull request merged**